### ISSUE TITLE
Better keyboard shortcuts

### ISSUE THREAD

**Issue by JonasDoebertin** [JonasDoebertin/kirby-visual-markdown#29]
Based on [this suggestion](https://twitter.com/_Mr_Spike_/status/597774678279757825) via Twitter, the keymaps should be changed to be actually useful.

![shortcuts](https://pbs.twimg.com/media/CEu45u4WAAARxpP.png)

**Comment by JonasDoebertin** [comment]
Suggestion for new shortcuts based on the Ghost shortcuts:

```
Cmd-H / Ctrl-H:          header1
Cmd-Alt-H / Ctrl-Alt-H:  header2
Cmd-B / Ctrl-B:          bold
Cmd-I / Ctrl-I:          italic
Ctrl-Q:                  blockquote
Ctrl-L:                  unorderedList
Ctrl-Alt-L:              orderedList
Ctrl-Alt-I:              image
Cmd-K / Ctrl-K':         link
```

**Comment by tweh** [comment]
+1 Looks nice!

**Comment by JonasDoebertin** [comment]
This is the current state:

![bildschirmfoto 2015-05-12 um 20 24 08](https://cloud.githubusercontent.com/assets/1367270/7595048/e656c9c6-f8e4-11e4-808e-726e64069216.png)

**Comment by JonasDoebertin** [comment]
Merged with the latest 1.3.2 release.
